infra/registry/network.tf

```diff
@@ -0,0 +1,43 @@
+resource "openstack_networking_network_v2" "network" {
+  name           = "network_1"
+  admin_state_up = true
+}
+
+resource "openstack_networking_subnet_v2" "subnet" {
+  name        = "subnet_1"
+  network_id  = openstack_networking_network_v2.network.id
+  cidr        = "172.16.98.0/23"
+  ip_version  = 4
+}
+
+resource "openstack_networking_router_v2" "router" {
+  name            = "router_"
```

Comment:
It's better if you set such resource names to the name of your terraform module. Like here
https://github.com/mekstack/mekstack/blob/009d31f75bde17f6dde9a0c3771dad6944e5e30b/infra/dnosha/network.tf#L7